microdata_tools/validation/components/unit_type_variables/KOMMUNE.json

```diff
@@ -11017,17 +10962,6 @@
         "validFrom": "1919-01-01",
         "validUntil": "2019-12-31"
       },
-      {
-        "code": "2100",
-        "categoryTitle": [
-          {
-            "languageCode": "no",
-            "value": "Ukjent kommune i Svalbard"
-          }
-        ],
-        "validFrom": "1919-01-01",
-        "validUntil": "2019-01-01"
-      },
```

Comment:
Will check to see if we should keep "ukjent kommune i Svalbard" and delete the other one.